internal/commands/vaultsecrets/vault_secrets.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package vaultsecrets
+
+import (
+	"github.com/hashicorp/hcp/internal/commands/vaultsecrets/apps"
+	"github.com/hashicorp/hcp/internal/pkg/cmd"
+	"github.com/hashicorp/hcp/internal/pkg/heredoc"
+)
+
+func NewCmdVaultSecrets(ctx *cmd.Context) *cmd.Command {
+	cmd := &cmd.Command{
+		Name:      "vault-secrets",
+		ShortHelp: "Manage Vault Secrets in Beta.",
```

Comment:
Should we adopt a style like `Manage Vault Secrets (in beta)` ?

Reply:
I liked that better too, but it'll fail the validation check w/ `short help text must start with a capital letter, end with a period, and contain only letters, apostrophes, hyphens, and spaces` :/

Reply:
Ahh, gotcha. In that case, let's remove beta from short help and move that under long help as you have.